docs/declarative-customization/site-theming/sharepoint-site-theming-overview.md

```diff
@@ -67,36 +67,34 @@ For the simplest experience, we recommend that you use modern themes with modern
 
 The modern site theming experience has been rolled out to classic site templates, too. While the new client-side theming architecture is more performant, if you have customizations on classic sites that aren’t rendering properly after you change the site’s theme, you can opt the site out of the new theming experience by disabling the feature. Please note, this opt-out only applies to classic sites where you have custom theme references that aren't rendering properly. By enabling this site-level opt-out you will disable the modern theming - and also lose the fast theming benefits it provides.
 
-To do this, you must use a Windows PowerShell script with a CSOM (client-side object model) wrapper. We recommend using the PnP enable feature command:
+To do this, you must use a Windows PowerShell script with a CSOM (client-side object model) wrapper. We recommend using the PnP PowerShell enable feature command:

```

Comment:
Why did you remove "PowerShell"? There's no "PnP enable feature"...

Reply:
I added PowerShell? It wasn't there before.

Reply:
Oops... my bad. I had my eyes crossed... #darn-monday